tests/test_install_mender_sh.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/python3
+# Copyright 2020 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        https://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import http.server
+import socketserver
+import threading
+import os
+import pytest
+
+SSERVER_PORT = 8000
+
+
+@pytest.fixture(scope="session")
+def script_server():
+    sdir = os.path.join(os.path.dirname(__file__), "..", "scripts")
+    os.chdir(sdir)
+
+    thread = None
+    with socketserver.TCPServer(
+        ("0.0.0.0", SSERVER_PORT), http.server.SimpleHTTPRequestHandler
+    ) as httpd:
+        httpd.allow_reuse_address = True
+        thread = threading.Thread(target=httpd.serve_forever)
+        thread.daemon = False
+        thread.start()
+        yield
+    httpd.shutdown()
+    thread.join()
+    print("cleaned up script server")
+
+
+class TestInstallMenderScript:
+    def _get_localhost_ip(self, setup_tester_ssh_connection):
+        """ We need to access the toplevel host's port to curl the script.
+            'localhost' of course doesn't work, but qemu runs on docker in 'net=host' mode, 
+            so the host's address is simply the default route.
+        """
+        result = setup_tester_ssh_connection.run(
+            "ip route | grep default | awk '{print $3}'"
+        )
+        return result.stdout.strip()
+
+    @pytest.mark.usefixtures("setup_test_container_f")
+    @pytest.mark.parametrize("channel", ["", "stable", "experimental"])
+    @pytest.mark.skip(
+        reason="no mender-shell in 'stable'; in 'experimental' depends on client 2.5.0 which is not there yet"
+    )
+    def test_default(
+        self,
+        script_server,
+        setup_tester_ssh_connection_f,
+        mender_dist_packages_versions,
+        mender_version,
+        channel,
+    ):
+        """ Default, no arg install installs mender-client and mender-shell (stable).
+        """
+        localhost = self._get_localhost_ip(setup_tester_ssh_connection_f)
+
+        if channel != "":
+            channel = "-c " + channel
+
+        setup_tester_ssh_connection_f.run(
+            "curl http://{}:{}/install-mender.sh | sudo bash -s -- {}".format(
+                localhost, SSERVER_PORT, channel
+            )
+        )
+
+        setup_tester_ssh_connection_f.run("dpkg -s mender-client")
+        setup_tester_ssh_connection_f.run("dpkg -s mender-shell")
```

Comment:
For better readability please use the full option
```suggestion
        setup_tester_ssh_connection_f.run("dpkg --status mender-client")
        setup_tester_ssh_connection_f.run("dpkg --status mender-shell")
```